spec/system/articles/moderator_moderates_an_article_spec.rb

```diff
@@ -17,14 +17,15 @@
     expect(page).to have_content(article.title)
   end
 
+  # rubocop:disable Rspec/Capybara/SpecificMatcher
   it "lets moderators visit /mod", js: true do
     visit "/#{user.username}/#{article.slug}/mod"
-
     expect(page).to have_selector('button[data-category="thumbsdown"][data-reactable-type="Article"]')
```

Comment:
I couldn't find a way to locate by `data` when using `have_button` and didn't want to spend too much time on this. Feel free to update if you know this. 
Tried `have_css`, but rubocop doesn't like it as well.